support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -542,7 +552,7 @@ function CheckoutComponent({ geoId }: Props) {
 			paymentFields = {
 				recaptchaToken: '',
 				paymentMethod: 'PayPal',
-				baid: formData.get('payPalBAID') as string,
```

Comment:
We could keep this reading from the form - your call, whatever feels nicer.

Reply:
Yes, changed